include/openthread/platform/logging.h

```diff
@@ -134,6 +134,7 @@ typedef enum otLogRegion
     OT_LOG_REGION_CLI      = 14, ///< CLI
     OT_LOG_REGION_CORE     = 15, ///< OpenThread Core
     OT_LOG_REGION_UTIL     = 16, ///< Utility module
+    OT_LOG_REGION_BLE      = 17, ///< Bluetooth LE module
```

Comment:
BLE stacks tend to be of sufficient complexity that dedicating a log region that can be enabled/disabled at will and easily filtered makes sense.  Also, the plan is to have TOBLE (core functionality) and BLE (platform layer) share this same region initially, so that may ease your concerns as well.